packages/sn-document-viewer-react/src/models/viewer-state.ts

```diff
@@ -68,4 +68,9 @@ export interface ViewerState {
    *
    */
   onPageChange: ObservableValue<number>
+
+  /**
+   *
+   */
+  visiblePages: number[]
```

Comment:
why visiblePages is needed at all? activePages is not good for this purpose?

Reply:
it will be an infinite loop :(